content/docs/music.md

```diff
@@ -5,17 +5,40 @@ next: "/docs/build"
 nextTitle: "Building Your Game"
 ---
 
-GB Studio is internally using [GBT Player](https://github.com/AntonioND/gbt-player). Additional documentation and sample MOD files can be found in its own repository.
+GB Studio is internally using [GBT Player](https://github.com/AntonioND/gbt-player) *(GameBoy Tracker)*. Additional documentation and sample MOD files can be found in its own repository.
 
 GBT Player reads and plays back .MOD files. You can use software such as [OpenMPT](https://openmpt.org/) or [MilkyTracker](https://milkytracker.titandemo.org/) to create .MOD tracker music. Of course, you can use other software that can edit .mod files (like [BassoonTracker](https://www.stef.be/bassoontracker/), [ProTracker](https://16-bits.org/pt.php), etc.)
 
+# How do I start?
+
+GameBoy music is made using a tracker, a *computer music* oriented program.
+
+1. Download a tracker (recommended [OpenMPT](https://openmpt.org/))
+2. In a blank Gbstudio project, find the file `assets/music/template.mod` and open it with your tracker.\
+ **You must edit this file for the right GameBoy instruments.**
+3. Start deleting notes from the song (some trackers support drag selection).
+4. Select your instument and press letter keys on the keyboard to make notes.
+5. Read your trackers documentation, and find tutorials on using trackers in general.
+6. Keep testing your song.mod in the GBstudio emulator, as it will always sound different.
```

Comment:
I think this section is really good but it just needs to be a bit more concise. Points 3 and 4 sound kinda obvious when the more important Point 5 just says to read its documentation, and other parts are vague

1. Download a tracker such as [OpenMPT](https://openmpt.org/))
2. Read your tracker's documentation and find some tutorials if you're not already familiar with how to use it.
3. In a blank GB Studio project, find the file `assets/music/template.mod` and open it with your tracker.\
 **You must edit this file to hear accurate GameBoy instruments in your tracker.**
4. Delete the song's contents but keep the instrument data. Some trackers support "zapping" the song.
5. Use the instrument list below to pick the sounds you want. Changing the samples in your tracker will not affect how they sound in-game.
6. Test your song.MOD in the GB Studio emulator often to keep track of any audible in-game differences.